### DISCUSSION TITLE
How can i work with server sent events(SSE) ?

### DISCUSSION THREAD

**Discussion by bvsvntv** [mistweaverco/kulala.nvim#330] [Q&A] [answered]
Most of the time i'm using kulala to work with `application/json` response type. Now i'm exploring other response types as well which includes [SSE](https://developer.mozilla.org/en-US/docs/Web/API/Server-sent_events/Using_server-sent_events), it has  `text/event-stream` response type. Is there any support for it ? If not then having that feature would be great.

**Comment by YaroSpace** [comment, marked as answer]
This is now available with `@accept chunked` directive.
